build.sbt

```diff
@@ -61,6 +61,9 @@ lazy val root = project.withId("spark-snowflake").in(file("."))
       // "org.apache.spark" %% "spark-sql-kafka-0-10" % "2.4.0",
       "org.apache.spark" %% "spark-core" % testSparkVersion % "provided, test",
       "org.apache.spark" %% "spark-sql" % testSparkVersion % "provided, test",
+      "org.apache.spark" %% "spark-core" % testSparkVersion % "provided, test" classifier "tests",
```

Comment:
I think the snow park is based on spark 3.0. But spark connector needs to support spark 2.3 and 2.4.
Do we need to port these test for spark 2.3/2.4? I think it may not be necessary. If we have to, it sounds a lot work for the porting.

Reply:
I'll test the changes with the other versions.

Reply:
aha, you figure out how to import test jar on SBT.

Reply:
Yes, that is a problem. We should decide to run such test only against spark 3.0 or all supported version (2.3 and 2.4). We may be not able to share test since they do change a lot.